service.twitterfeed/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.twitterfeed" name="Twitter Feed" version="0.8.28" provider-name="L0RE">
+    <requires>
+        <import addon="xbmc.python" version="2.19.0"/>
```

Comment:
jarvis is 2.24.0 ... just check the versions here: http://kodi.wiki/view/addon.xml